spec/support/shared_examples/flow.rb

```diff
@@ -0,0 +1,42 @@
+# TS, 7/23/21: I think we should deprecate this file and move the testing logic into each
+# feature spec, since each page is slightly different and there are different things we want to test
+# in each one. I also don't think we need to be testing the change division flow for every module.
```

Comment:
I was wondering about this and this also sounds good to me as a direction to go in.